### ISSUE TITLE
support new file-level constants

### ISSUE THREAD

**Issue by cruzdanilo** [protofire/solhint#261]
solidity v0.7.4 supports file-level constants, which causes solhint parsing error.

https://solidity.ethereum.org/2020/10/19/solidity-0.7.4-release-announcement/

**Comment by gitpusha** [comment]
yes, I get this errror for this sol code 

```solidity
// SPDX-License-Identifier: UNLICENSED
pragma solidity 0.7.4;

import {
    IGelatoGasPriceOracle
} from "../interfaces/gelato/IGelatoGasPriceOracle.sol";

IGelatoGasPriceOracle constant GELATO_GAS_PRICE_ORACLE = IGelatoGasPriceOracle(
    0x169E633A2D1E6c10dD91238Ba11c4A708dfEF37C
);
```

`  10:0  error  Parse error: extraneous input 'IGelatoGasPriceOracle' expecting {<EOF>, 'pragma', 'import', 'abstract', 'contract', 'interface', 'library', 'struct', 'function', 'enum', 'constructor', 'fallback', 'receive'}`

**Comment by fvictorio** [comment]
This should be fixed in `solhint@3.3.2`, BUT I didn't update any rule, so I'd suggest you check if your expectations about contract-level constants match what happens with file-level constants (for example, if naming works). If something that works for contract-level constants but doesn't (and should) for file-level constants, please open an issue!
